firestore-bigquery-export/scripts/import/src/config.ts

```diff
@@ -0,0 +1,188 @@
+import * as inquirer from "inquirer";
+import * as program from "commander";
+
+import { CliConfig } from "./types";
+
+const BIGQUERY_VALID_CHARACTERS = /^[a-zA-Z0-9_]+$/;
+const FIRESTORE_VALID_CHARACTERS = /^[^\/]+$/;
+
+const FIRESTORE_COLLECTION_NAME_MAX_CHARS = 6144;
+const BIGQUERY_RESOURCE_NAME_MAX_CHARS = 1024;
+
+const validateBatchSize = (value: string) => {
+  return parseInt(value, 10) > 0;
+};
+
+const validateLocation = (value: string) => {
+  const index = [
+    "us-west4",
+    "us-west2",
+    "northamerica-northeast1",
+    "us-east4",
+    "us-west1",
+    "us-west3",
+    "southamerica-east1",
+    "us-east1",
+    "europe-west1",
+    "europe-north1",
+    "europe-west3",
+    "europe-west2",
+    "europe-west4",
+    "europe-west4",
+    "europe-west6",
+    "asia-east2",
+    "asia-southeast2",
+    "asia-south1",
+    "asia-northeast2",
+    "asia-northeast3",
+    "asia-southeast1",
+    "australia-southeast1",
+    "asia-east1",
+    "asia-northeast1",
+    "us",
+    "eu",
+  ].indexOf(value.toLowerCase());
+
+  return index !== -1;
+};
+
+const validateInput = (
+  value: string,
+  name: string,
+  regex: RegExp,
+  sizeLimit: number
+) => {
+  if (!value || value === "" || value.trim() === "") {
+    return `Please supply a ${name}`;
+  }
+  if (value.length >= sizeLimit) {
+    return `${name} must be at most ${sizeLimit} characters long`;
+  }
+  if (!value.match(regex)) {
+    return `The ${name} must only contain letters or spaces`;
```

Comment:
if regex checks for something besides letters/spaces, this warning may be misleading

Reply:
Updated error message.